test/main.js

```diff
@@ -12,7 +12,12 @@ const autoprefixer = require('autoprefixer');
 const tap = require('gulp-tap');
 const globule = require('globule');
 
-const sass = require('../index')(require('node-sass'));
+const COMPILER = process.env.SASS_COMPILER === 'sass' ? 'sass' : 'node-sass';
+
+// eslint-disable-next-line import/no-dynamic-require
+const gulpSass = require('../index')(require(COMPILER));
```

Comment:
FWIW reducing unnecessary diff noise lowers the burden on reviewers.

```suggestion
const sass= require('../index')(require(COMPILER));
```

Reply:
I'm aware but initially I was planning to use a `sass` variable for dart sass. I guess now I can revert this change.